Tests/UnitTests/Core/Utilities/BoundingBoxTest.cpp

```diff
@@ -485,9 +487,9 @@ BOOST_AUTO_TEST_CASE(frustum_intersect) {
     // ActsVector<BoundingBoxScalar,2> dir    = {1, 0};
     // ActsVector<BoundingBoxScalar,2> origin = {min + step * i, min + step *
     // j}; origin.x() *= 1.10;  // visual Eigen::Rotation2D<BoundingBoxScalar>
-    // rot(2 * M_PI / static_cast<BoundingBoxScalar>(n) * i); BoundingBoxScalar
-    // angle = 0.5 * M_PI / n * j; Frustum2                 fr(origin, rot *
-    // dir, angle); fr.svg(os, w, w, 2);
+    // rot(2 * std::numbers::pi / static_cast<BoundingBoxScalar>(n) * i);
```

Comment:
Probably should check why this code is commented.

Reply:
This just returns an svg, which could be investigated manually. I try to reactivate it that it doesn't become disfunctional.

Reply:
I will reactivated the commented out stuff here in a future PR:
- https://github.com/acts-project/acts/pull/3829